app/src/main/res/layout-land/support_error_layout.xml

```diff
@@ -5,7 +5,7 @@
     android:id="@+id/generic_purchase_error_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/background_content_payments"
+    android:background="@color/styleguide_blue"
```

Comment:
This file is common to the payment flow, it needs to be light mode. I suggest creating a new one for dark mode